vipps-login-api.md

```diff
@@ -725,69 +725,99 @@ Web-views should not be used when using Vipps Login in a native application. Ins
 ``` 
 merchant-app://callback/?state=RFiQdrl_lvJUpVmTRSKmsZRGLM0G1N1qh0WebZ1gDNk&resume_uri=https%3A%2F%2Fapi.vipps.no%2Fvipps-login-idp%2Findex.html%3FtabId%3D7607f7f0-7ae2-49b7-9cb5-102143dac4ea
 ```
 
-A typical flow/implementation might look like this:
-1. An OpenID authentication flow authorization URI is generated on the merchant backend. The URI is communicated to the merchant app and is used to initiate Vipps login in an external browser that is opened within the app, see [specification](#using-vipps-login-in-native-applications)for details.
-2. If required, Vipps login will deeplink the user over to the Vipps app (if the user is remembered in the browser the user will be authenticated directly - the user will then be on step 5 below)
-3. When the user has approved the login in the Vipps app the `app_callback_uri` is triggered.
-4. The `app_callback_uri` will return the user to the merchant app. The app now has 2 separate choices.
+#### A typical flow/implementation might look like this:
+![](images/app2appsuggestion.png)
+1. An [OpenID authentication flow authorization URI](#oauth-20-authorize) is generated on the merchant backend. 
+   The URI is communicated to the merchant app
+2. and is used to initiate Vipps login in an external browser that is opened within the app, 
+   see [specification](#using-vipps-login-in-native-applications) for details.
+3. If required, Vipps login will deeplink the user over to the Vipps app 
+   (if the user is remembered in the browser the user will be authenticated directly 
+    - the user will then be on step 6 below)
+4. When the user has approved the login in the Vipps app redirects the user to the `app_callback_uri`.
+5. The `app_callback_uri` should return the user to the merchant app. The app now has 2 separate choices.
    * Display the same browser instance that was created in 1.
-   * Open a browser again using the `resume_uri`query parameter that is returned with the request to the `app_callback_uri`.
-5. Vipps login will finalise the authentication of the user and acquire consent to share information if needed. When this is finished the user will be redirected to the `redirect_uri`. The Vipps login process has now finished and the merchant controls the remaining process.
-6. The merchant app sends the `code` and `state`parameters received in the callback to the merchant backend
-7. The merchant backend fetches the access tokens and user information
-
-Examples:
+   * Open a browser again using the `resume_uri`-query parameter that is returned with the request to the `app_callback_uri`.
+6. Vipps login will finalise the authentication of the user and acquire consent to share information if needed. 
+   When this is finished the user will be redirected to the `redirect_uri`. 
+   The Vipps login process has now finished and the merchant controls the remaining process.
+7. The merchant app sends the `code` and `state`parameters received in the callback to the merchant backend
```

Comment:
Have to decide on "The merchant app" vs "Merchant app" if we go with the suggested pattern.
```suggestion
7. The merchant app sends the `code` and `state`parameters received in the callback to the merchant backend
```